scss/_common.scss

```
@@ -0,0 +1,87 @@
+.button {
+  font-family: "Aleo", serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  letter-spacing: 0.07em;
+  text-align: center;
+  height: 60px;
+  width: 213px;
+  border-radius: 0px;
+  background: url(../img/Button.svg) center center no-repeat;
+  color: rgba(255, 255, 255, 1);
+  text-transform: uppercase;
+  &:hover {
+    opacity: 0.9;
+  }
+  &:active {
+    border: 2px solid rgb(243, 245, 255);
+  }
+}
+
+.h2 {
+  font-family: "Aleo", serif;
+  font-size: 36px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 40px;
+  letter-spacing: 0em;
+  text-align: left;
+  color: rgba(255, 255, 255, 1);
+}
+
+.h3 {
+  font-family: "Josefin Sans", sans-serif;
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 28px;
+  letter-spacing: 0.1em;
+  color: rgba(255, 255, 255, 1);
+}
+
+.h4 {
+  font-family: "Aleo", serif;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  letter-spacing: 0.1em;
+  color: rgba(255, 255, 255, 1);
+  text-transform: uppercase;
+}
+
+.h5 {
+  font-family: "Josefin Sans", sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0em;
+  text-align: left;
+  color: rgba(255, 255, 255, 1);
+  text-transform: uppercase;
+}
+
+.h6 {
+  font-family: "Josefin Sans", sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 300;
+  line-height: 28px;
+  letter-spacing: 0.1em;
+  text-align: left;
+  color: rgba(255, 255, 255, 1);
+}
+
+.p {
+  font-family: "Josefin Sans", sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 300;
+  line-height: 28px;
+  letter-spacing: 0.1em;
+  text-align: left;
+  color: rgba(255, 255, 255, 1);
```

это значения по умолчанию, так что можно опустить
```suggestion
  font-family: "Josefin Sans", sans-serif;
  font-size: 16px;
  font-weight: 300;
  line-height: 28px;
  letter-spacing: 0.1em;
  color: rgba(255, 255, 255, 1);
```